turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -920,9 +912,9 @@ pub(crate) async fn analyse_ecmascript_module_internal(
                 kind,
                 ast_path: condition_ast_path,
                 span: _,
-                in_try,
+                in_try: _,
```

Comment:
Looks like we have information about whether we are in try or not. Why shouldn't we use it?